services/discordService.js

```diff
@@ -51,8 +51,25 @@ const addRoleToUser = async (userid, roleid) => {
   return response;
 };
 
+const removeRoleFromUser = async (roleId, discordId) => {
+  try {
+    const authToken = await generateAuthTokenForCloudflare();
+    const data = await fetch(`${DISCORD_BASE_URL}/roles`, {
+      method: "DELETE",
+      headers: { "Content-Type": "application/json", Authorization: `Bearer ${authToken}` },
+      body: JSON.stringify({ userid: discordId, roleid: roleId }),
+    });
+    const response = await data.json();
+    return response;
+  } catch (err) {
+    logger.error("Error in making fetch call to remove the role", err);
```

Comment:
I think having a general message is better as error can occur due to any other operation as well and not only due to fetch